pyne/ace.pyx

```diff
@@ -331,6 +333,8 @@ class Library(object):
                 temp_in_K = round(temp * 1e6 / 8.617342e-5)
                 print("Loading nuclide {0} at {1} K".format(name, temp_in_K))
             self.tables[name] = table
+            if name_old:
```

Comment:
This case will also be skipped if `name_old` is a str and has length 0.  Is this desirable?